reference/test/preconditioner/isai_kernels.cpp

```diff
@@ -47,11 +47,11 @@ OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
 #include <ginkgo/core/matrix/csr.hpp>
 #include <ginkgo/core/matrix/dense.hpp>
 #include <ginkgo/core/preconditioner/ilu.hpp>
-#include <ginkgo/core/preconditioner/isai.hpp>
 
 
 #include "core/test/utils.hpp"
 #include "matrices/config.hpp"
+#include "reference/preconditioner/isai_kernels.cpp"
```

Comment:
Is including the `.cpp` correct here? Surely not?

Reply:
Oh yeah, good catch. It was probably working before because the .cpp file includes the .hpp file, but still